moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -145,11 +145,11 @@ const ProjectActivityLog = () => {
         event.record_type === "moped_project" &&
         event.operation_type === "INSERT"
       ) {
-        // Move it to the top of the list (make it first)
-        outputList.unshift(event);
-      } else {
-        // Else, just stack it to the bottom
+        // Append it to the end of the list (make it last)
         outputList.push(event);
+      } else {
+        // Else, prepend it to the beginning of the list (so it appears in descending chronological order)
+        outputList.unshift(event);
```

Comment:
This function seemed to be straying from its intended purpose as documented and it seemed a bit janky to me to be ordering activity log items when we could seem to control that through reliably the query. So I made a PR using that query based approach instead of in the UI. Let me know what you think: https://github.com/cityofaustin/atd-moped/pull/283/files

Reply:
@mateoclarke I reworked the PR to order the query results in descending order, then modified the operations on the frontend to focus only on pulling out the creation event and adding it to the end of the event list that is displayed. I tried to ensure this operation would make sense and be easily readable to other devs. Please let me know if this looks good!